src/WebJobs.Extensions.EasyTables/Bindings/EasyTableItemValueBinder.cs

```diff
@@ -0,0 +1,149 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Reflection;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Host.Bindings;
+using Microsoft.WindowsAzure.MobileServices;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Extensions.EasyTables
+{
+    internal class EasyTableItemValueBinder<T> : IValueBinder
+    {
+        private ParameterInfo _parameter;
+        private EasyTableContext _context;
+        private string _id;
+        private JObject _originalItem;
+
+        public EasyTableItemValueBinder(ParameterInfo parameter, EasyTableContext context, string id)
+        {
+            _parameter = parameter;
+            _context = context;
+            _id = id;
+        }
+
+        public Type Type
+        {
+            get { return _parameter.ParameterType; }
+        }
+
+        public async Task SetValueAsync(object value, CancellationToken cancellationToken)
+        {
+            if (value == null || _originalItem == null)
+            {
+                return;
+            }
+
+            await SetValueInternalAsync(_originalItem, value, _context);
+        }
+
+        public object GetValue()
+        {
+            object item = null;
+
+            if (typeof(T) == typeof(JObject))
+            {
+                IMobileServiceTable table = _context.Client.GetTable(_context.ResolvedTableName);
+                IgnoreNotFoundException(() =>
+                {
+                    item = table.LookupAsync(_id).Result;
+                    _originalItem = CloneItem(item);
+                });
+            }
+            else
+            {
+                // treat this as POCO
+                IMobileServiceTable<T> table = _context.Client.GetTable<T>();
+                IgnoreNotFoundException(() =>
+                {
+                    item = table.LookupAsync(_id).Result;
+                    _originalItem = CloneItem(item);
+                });
+            }
+
+            return item;
+        }
+
+        public string ToInvokeString()
```

Comment:
Have you looked at your invocations in the Dashboard to see what UI strings you emit and what makes sense for your binding?